FreeRTOS/Test/CMock/queue/queue_utest.c

```diff
@@ -36,39 +36,41 @@
 
 /* Test includes. */
 #include "unity.h"
+#include "unity_memory.h"
 
 /* Mock includes. */
 #include "mock_task.h"
 #include "mock_list.h"
+#include "mock_fake_assert.h"
 
 /* ============================  GLOBAL VARIABLES =========================== */
-static uint16_t usMallocFreeCalls = 0;
 
 /* ==========================  CALLBACK FUNCTIONS =========================== */
 
 void * pvPortMalloc( size_t xSize )
 {
-    return malloc( xSize );
+    return unity_malloc( xSize );
 }
 void vPortFree( void * pv )
 {
-    return free( pv );
+    return unity_free( pv );
 }
 
 /*******************************************************************************
  * Unity fixtures
  ******************************************************************************/
 void setUp( void )
 {
+    vFakeAssert_Ignore();
```

Comment:
Does it mean that we are ignoring asserts for all tests? Or would we use other mocked functions to test that the assert does get triggered in negative tests?

Reply:
Yes. The current unit tests in this repository ignore configASSERT.

I have a prototype of "catching" configASSERTs, but need to clean it up and submit that with the queue utest.

I will need to add something more complex like [c_exception](https://github.com/ThrowTheSwitch/CException/) to cover cases where returning from a configASSERT would result in a segfault. Otherwise, we will just need to avoid cases where that would occur.